general/envsetup/envsetup.go

```diff
@@ -197,8 +205,36 @@ func (ftc *EnvSetupCommand) setupInvitedUser() (server *config.ServerDetails, er
 		return
 	}
 	if server.Password == "" {
+		err = ftc.scanAndValidateJFrogPasswordFromConsole(server)
+	}
+	return
+}
+
+func (ftc *EnvSetupCommand) scanAndValidateJFrogPasswordFromConsole(server *config.ServerDetails) (err error) {
+	// User has limited number of retries to enter his correct password.
+	// Password validation is operated by Artifactory EncryptedPassword API.
+	for i := 1; i <= enterPasswordMaxRetries; i++ {
 		server.Password, err = ioutils.ScanJFrogPasswordFromConsole()
+		if err != nil {
+			return
+		}
+		server.ArtifactoryUrl = clientutils.AddTrailingSlashIfNeeded(server.Url) + "artifactory/"
+		var artAuth auth.ServiceDetails
+		artAuth, err = server.CreateArtAuthConfig()
+		if err != nil {
+			return
+		}
+		_, err = utils.GetEncryptedPasswordFromArtifactory(artAuth, false)
+		if err != nil {
+			if i != enterPasswordMaxRetries {
+				log.Info("wrong password! please try again. ")
+			}
+			continue
+		}
+		// No error while encrypting password => valid password.
+		return
```

Comment:
Can be simplified to this:
```suggestion
		_, err = utils.GetEncryptedPasswordFromArtifactory(artAuth, false)
		if err == nil {
			// No error while encrypting password => valid password.
			return
		}
		log.Info("wrong password! please try again. ")
```